data/json/mapgen/parking_garage4x4.json

```
@@ -0,0 +1,415 @@
+        "|                                           |***",
+        "|                                           =***",
+        "|                                           =***",
+        "W                                           =***",
+        "W-----               -                 -----=***",
+        "W                    -                      =***",
+        "W                    -                  ssss|***",
+        "|                    -                  s/si|***",
+        "||||||||||||====|====|====|====|====|====||||***",
+        "************************************************"
+      ],
+      "palettes": [ "parking_garage4x4_palette" ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 3, 17 ], "y": [ 13, 15 ], "chance": 20 } ],
+      "place_vehicles": [
+        { "vehicle": "parking_garage", "x": 18, "y": 5, "chance": 25, "rotation": 270 },
+        { "vehicle": "parking_garage", "x": 28, "y": 5, "chance": 25, "rotation": 270 },
+        { "vehicle": "parking_garage", "x": 33, "y": 5, "chance": 25, "rotation": 270 },
+        { "vehicle": "parking_garage", "x": 38, "y": 5, "chance": 25, "rotation": 270 },
+        { "vehicle": "parking_garage", "x": 19, "y": 18, "chance": 25, "rotation": 90 },
+        { "vehicle": "parking_garage", "x": 29, "y": 18, "chance": 25, "rotation": 90 },
+        { "vehicle": "parking_garage", "x": 34, "y": 18, "chance": 25, "rotation": 90 },
+        { "vehicle": "parking_garage", "x": 39, "y": 18, "chance": 25, "rotation": 90 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ [ "parking_garage4x4_2_0", "parking_garage4x4_2_1" ], [ "parking_garage4x4_2_2", "parking_garage4x4_2_3" ] ],
+    "weight": 250,
+    "object": {
+      "fill_ter": "t_thconc_floor",
+      "rows": [
+        "||||||||||||====|====|====|====|====|====||||***",
+        "|EE|s                                       |;;*",
+        "=EE!s                                       |iz2",
+        "=||es                                       =i;*",
+        "=ssss                                       =/;*",
+        "=                                           =;;*",
+        "|                  zzzzzzzzzzzzzzz          =***",
+        "|                zz               zz        =***",
+        "|-----          zz                  z       |***",
+        "|              zz                   z       |***",
+        "|              z                    z       |***",
+        "|              z                    z       |***",
+        "|              z                    z       |***",
+        "|                       sssss       z       |***",
+        "|                       sIsIs       z       |***",
+        "|-----                  sssss               |***",
+        "|                                           |***",
+        "|                                    >>>>>>>|***",
+        "|                             sssss  OOOOOOO|***",
```

```suggestion
        "|                                           |***",
        "|                             sssss         |***",
```
This is the topmost floor, no need for a ramp leading up here.